pylutron/__init__.py

```diff
@@ -394,12 +394,14 @@ def _parse_cci(self, keypad, component_xml):
   def _parse_led(self, keypad, component_xml):
     """Parses an LED device that part of a keypad."""
     component_num = int(component_xml.get('ComponentNumber'))
+    
     led_base = 80
     if keypad.type == 'MAIN_REPEATER':
       led_base = 100
     led_num = component_num - led_base
+    name = f"keypad {keypad.id}: LED {led_num}"
```

Comment:
ditto

Reply:
see above

Reply:
modified. HA has to compose the unique name